app/Models/Events/Meeting.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Models\Events;
+
+use App\Models\User\User;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\Relations\HasOne;
+
+class Meeting extends Model
+{
+    use HasFactory;
+
+    protected $table = 'meetings';
+
+    protected $fillable = [
+        'content',
+        'meeting_type_id',
+        'user_created_id',
```

Comment:
Acho que poderia ficar mais sucinto e explicativo com algo do tipo `author_id` ou `admin_id`. 
```suggestion
        'admin_id',
```